src/backend/API/Events.ts

```diff
@@ -35,7 +38,21 @@ export class Events {
   }
 
   getRawEventsFromGroup(group: string): Promise<Array<any>> {
-    const url = urljoin(group, 'events', `?has_ended=false&no_later_than=${getISODate(this.getNoLaterDate())}`);
+    const params : {[index: string]:any} = {
+      has_ended: 'false',
+      no_earlier_than: getISODate(this.startDate),
+      no_later_than: getISODate(this.getNoLaterDate(this.startDate))
+    };
+
+    let params_string = '?';
+    for (const param in params) {
+      if (params.hasOwnProperty(param)) {
```

Comment:
eh, where was it... I have written comment on this one, eh.
primo, Object.entries is better than for-in here
secundo, https://www.npmjs.com/package/query-string will solve this problem instead